chain/src/tests/basic.rs

```diff
@@ -177,17 +178,17 @@ fn test_transaction_spend_in_same_block() {
         shared
             .lock_chain_state()
             .cell(&OutPoint::new_cell(tx2_hash.to_owned(), 0)),
-        CellStatus::live_cell(
-            CellMetaBuilder::default()
-                .out_point(CellOutPoint {
-                    tx_hash: tx2_hash.to_owned(),
-                    index: 0
-                })
-                .data_hash(tx2_output.data_hash())
-                .capacity(tx2_output.capacity)
-                .block_info(BlockInfo::new(parent_number4, 0, parent_hash4))
-                .build()
-        )
+        CellStatus::live_cell(CellMeta {
```

Comment:
why change this from `CellMetaBuilder::builder` to `CellMeta {}`? Add a new method `data_bytes` in `CellMetaBuilder`?

Reply:
We remove `data_bytes` method by intention, forbidden to set `data_bytes` can force data to be consistent.